Content.Client/Storage/Systems/ItemMapperSystem.cs

```diff
@@ -41,13 +43,14 @@ private void InitLayers(ItemMapperComponent component, SpriteComponent spriteCom
         if (!appearance.TryGetData<ShowLayerData>(StorageMapVisuals.InitLayers, out var wrapper))
             return;
 
+        TryComp<StorageFillComponent>(component.Owner, out var storageFill);
         component.SpriteLayers.AddRange(wrapper.QueuedEntities);
 
         foreach (var sprite in component.SpriteLayers)
         {
             spriteComponent.LayerMapReserveBlank(sprite);
             spriteComponent.LayerSetSprite(sprite, new SpriteSpecifier.Rsi(component.RSIPath!, sprite));
-            spriteComponent.LayerSetVisible(sprite, false);
+            spriteComponent.LayerSetVisible(sprite, storageFill != null);
```

Comment:
what does this even fix? this feels more like an init order bug than anything else (storagefill not spawning before itemmapper does). tying this behavior specifically to storagefill feels like it is going to introduce even more bugs

Reply:
I'm back after a two week absence. While playing last night I noticed other issues with placing items into the belt weren't updating the worn belt's display. I'll play with it some more to see if there's alternatives to fixing this or if changing it here is necessary.

Reply:
Yeah, it's not addressing this other issue I'm noticing. I'll take a different approach and see if I can fix both issues.